idunn/geocoder/nlu_client.py

```diff
@@ -204,6 +204,11 @@ async def get_intentions(self, text, lang, focus=None):
         tags_list = [t for t in response_nlu.json()["NLU"] if t["tag"] != "O"]
 
         if self.is_poi_request(tags_list):
+            logger.info(
+                "Detected POI request for '%s'",
+                text,
+                extra={"lang": lang, "focus": focus, "tags_list": tags_list},
```

Comment:
`text` would be useful in the `extra` object too, to make the extraction easier.

Also, all keys in `extra` appear in the log message at the root level (when using json format for logs).
I think it would be more appropriate to nest this data under a dedicated key, such as `intention_detection` ?

`focus` is a shapely `Point`, that should be serialized in an explicit way.